buildpacks/nodejs-npm-install/buildpack.toml

```diff
@@ -0,0 +1,28 @@
+api = "0.9"
+
+[buildpack]
+id = "heroku/nodejs-npm-install"
+version = "1.1.6"
+name = "Heroku Node.js npm Install"
+homepage = "https://github.com/heroku/buildpacks-nodejs"
+keywords = ["node", "node.js", "nodejs", "javascript", "js", "npm", "install"]
+
+[[buildpack.licenses]]
+type = "MIT"
+
+[[stacks]]
+id = "*"
+
+[[stacks]]
+id = "heroku-20"
+
+[[stacks]]
+id = "heroku-22"
+
+[[stacks]]
+id = "io.buildpacks.stacks.bionic"
+
```

Comment:
The wildcard stack makes these other stacks redundant (the pack CLI bug with wildcard stacks was fixed long enough ago we no longer need to work around it)

```suggestion
```